### PR TITLE
document transactionOptions in PrismaClient constructor

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 1]
```diff
@@ -407,65 +407,49 @@ const adapter = new PrismaNeon(pool)
 const prisma = new PrismaClient({ adapter })
 ```
 
-### <inlinecode>rejectOnNotFound</inlinecode>
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]: We have users on older versions than Prisma 5 (and probably also links to this section publicly and in our docs). We should keep it until we have another solution to version our documentation.

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 2]
```diff
@@ -407,65 +407,49 @@ const adapter = new PrismaNeon(pool)
...
 
 </Admonition>
 
-Use the `rejectOnNotFound` parameter to configure `findUnique` and/or `findFirst` to throw an error if the record was not found. By default, both operations return `null` if the record is not found.
+Allows to set [transaction options](/orm/prisma-client/queries/transactions#transaction-options) globally on the constructor level.
 
 #### Remarks
 
-- You can configure `rejectOnNotFound` on a per-request level for both [`findUnique`](#findunique) and [`findFirst`](#findfirst)
+- The transaction levels can be override on a per-transaction level.
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]:
```suggestion
- The transaction levels can be overriden on a per-transaction level.
```

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 3]
```diff
@@ -407,65 +407,49 @@ const adapter = new PrismaNeon(pool)
...
+  transactionOptions: {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable,
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  },
 })
 ```
 
-##### Enable globally for a specific operation
+### <inlinecode>adapter</inlinecode>
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx:444]: Nice along-the-way addition.

File: content/200-orm/500-reference/050-prisma-client-reference.mdx [hunk 4]
```diff
@@ -407,65 +407,49 @@ const adapter = new PrismaNeon(pool)
...
-    findUnique: {
-      User: (err) => new Error('User error'),
-      Post: (err) => new Error('Post error!'),
-    },
-  },
-})
-```
+This is available from version 5.4.0 and newer behind the `driverAdapters` feature flag.
+
+</Admonition>
```
Comment (janpio) [inline content/200-orm/500-reference/050-prisma-client-reference.mdx]: Nice along-the-way addition.

**Review by janpio — CHANGES_REQUESTED** [review]
see comment

File: content/200-orm/200-prisma-client/000-setup-and-configuration/015-instantiate-prisma-client.mdx [hunk 5]
```diff
@@ -35,7 +35,7 @@ const prisma = new PrismaClient()
 
 <Tip>
 
-You can further customize `PrismaClient` with [constructor parameters](/orm/reference/prisma-client-reference#prismaclient) - for example, set [logging levels](/orm/prisma-client/observability-and-logging/logging) or customize [error formatting](error-formatting).
+You can further customize `PrismaClient` with [constructor parameters](/orm/reference/prisma-client-reference#prismaclient) — for example, set [logging levels](/orm/prisma-client/observability-and-logging/logging), [transaction options](/orm/prisma-client/queries/transactions#interactive-transactions) or customize [error formatting](error-formatting).
```
Comment (nikolasburk) [inline content/200-orm/200-prisma-client/000-setup-and-configuration/015-instantiate-prisma-client.mdx]:
```suggestion
You can further customize `PrismaClient` with [constructor parameters](/orm/reference/prisma-client-reference#prismaclient) — for example, set [logging levels](/orm/prisma-client/observability-and-logging/logging), [transaction options](/orm/prisma-client/queries/transactions#transactions-options) or customize [error formatting](error-formatting).
```

File: content/200-orm/500-reference/050-prisma-client-reference.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/200-orm/500-reference/050-prisma-client-reference.mdx]: Fair enough! Re-added this section ✅

File: content/200-orm/200-prisma-client/100-queries/058-transactions.mdx [hunk 6]
```diff
@@ -311,6 +317,18 @@ await prisma.$transaction(
...
+const prisma = new PrismaClient({
+  transactionOptions: {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable,
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  },
+})
+```
+
 <Admonition type="warning">
```
Comment (janpio) [inline content/200-orm/200-prisma-client/100-queries/058-transactions.mdx]: This warning now feels a bit out of plce under "Transaction Option". Does it need to be moved up maybe?

**Review by janpio — APPROVED** [review]
✅ 

(see last comment for nit)

File: content/200-orm/200-prisma-client/100-queries/058-transactions.mdx (hunk 6 above)
Comment (nikolasburk) [reply content/200-orm/200-prisma-client/100-queries/058-transactions.mdx]: Good catch! Moved it to the first section before the example 👍

**Pull request merged**